README.md

```diff
@@ -534,7 +534,7 @@ the COOL environment.
 | samba\_server\_security\_group | The security group for the Samba file share server instances. |
 | scanner\_security\_group | A security group that should be applied to all instance types that perform scanning.  This security group allows egress to anywhere as well as ingress from anywhere via ICMP. |
 | ssm\_session\_role | An IAM role that allows creation of SSM SessionManager sessions to any EC2 instance in this account. |
-| teamserver\_instance\_profile | The instance profile for the Teamserver instances. |
+| teamserver\_instance\_profile | The instance profiles for the Teamserver instances. |
```

Comment:
See previous comment in this same file.

Reply:
👍   See https://github.com/cisagov/cool-assessment-terraform/pull/153/commits/80a13efd98e839e727fc46b81bc8a35fb4d421fc.